src/udiskslinuxpartition.c

```diff
@@ -289,62 +289,31 @@ udisks_linux_partition_update (UDisksLinuxPartition   *partition,
 
 /* ---------------------------------------------------------------------------------------------------- */
 
-static gboolean
-handle_set_flags (UDisksPartition       *partition,
-                  GDBusMethodInvocation *invocation,
-                  guint64                flags,
-                  GVariant              *options)
+typedef gboolean (*_set_common_callback) (GDBusMethodInvocation *invocation,
+                                          UDisksBlock           *block,
+                                          gchar                 *device_name,
+                                          UDisksPartitionTable  *partition_table,
+                                          gchar                 *partition_name,
+                                          UDisksBaseJob         *job,
+                                          void                  *user_data);
+
+typedef void (*_partition_complete_set) (UDisksPartition        *object,
+                                         GDBusMethodInvocation  *invocation);
+
+
+static gboolean _set_flags (GDBusMethodInvocation *invocation,
+                            UDisksBlock           *block,
+                            gchar                 *device_name,
+                            UDisksPartitionTable  *partition_table,
+                            gchar                 *partition_name,
+                            UDisksBaseJob         *job,
+                            void                  *user_data)
```

Comment:
No need to start identifiers with ``_``. And the above ``typedef`` definitions define types so the identifiers should be CamelCase.